Jenkinsfile

```diff
@@ -1 +1 @@
-buildPlugin(platforms: ['linux'], jdkVersions: [11])
+buildPlugin(platforms: ['linux'], jdkVersions: [11])
```

Comment:
We can probably remove this from the diff of this PR, the only difference from `master` is the newline at the end.